src/core/api/checkout/checkout_ecommerce/v1/_openapi.json.tpl

```diff
@@ -1448,6 +1448,14 @@
                 "example": "www.comune.di.prova.it/pagopa/error.html"
               }
             }
+          },
```

Comment:
Are we still updating this api group? As I know this one is marked for removal waiting for frontend upgrade in UAT, so I will not upgrade also this API group upgrading only the one into ecommerce-app domain.
What do you think?

Reply:
You're right. I updated to be 100% compliant

Reply:
Removed